src/game/management.rs

```diff
@@ -0,0 +1,315 @@
+use std::thread;
+use std::sync::mpsc::{self,Sender};
+use std::error::Error as StdError;
+
+use mio::Sender as MioSender;
+use serde_json::ser::to_vec_pretty;
+use serde::Serialize;
+use uuid::Uuid;
+use iron::prelude::*;
+use iron::status::Status;
+use iron::headers::ContentType;
+use iron::{BeforeMiddleware};
+use iron::error::HttpError;
+use bodyparser::Struct;
+use router::{Router};
+use plugin::Extensible;
+use modifier::Modifier;
+use mount::Mount;
+
+use lycan_serialize::AuthenticationToken;
+
+use id::{Id,WeakId};
+use messages::Request as LycanRequest;
+use messages::Command;
+use data::{ConnectCharacterParam,Map,GetInstances,GetMaps};
+use entity::Entity;
+use instance::management::*;
+use game::Game;
+
+// TODO
+// - Set correct headers in all responses
+// - Check if correct heahers are set (e.g. Content-Type)
+
+pub fn start_management_api(sender: MioSender<LycanRequest>) {
+    thread::spawn(move || {
+        let router = create_router(sender);
+        let mut mount = Mount::new();
+        mount.mount("/api/v1", router);
+        let mut chain = Chain::new(mount);
+        chain.link_before(AuthenticationMiddleware("abcdefgh".to_string()));
```

Comment:
Yes it is the authentication token, I can add it to the readme. It already is by default in all the scripts I have written.